crates/as-if-std/Cargo.toml

```diff
@@ -2,7 +2,8 @@
 name = "as-if-std"
 version = "0.1.0"
 authors = ["Alex Crichton <alex@alexcrichton.com>"]
-edition = "2018"
+edition = "2021"
+rust-version = "1.58.0"
```

Comment:
I would strongly prefer we do not document rust-version in a bunch of example crates if we're just going to pointlessly churn it.
```suggestion
```